<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>636</width>
    <height>703</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>20</y>
     <width>381</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial</family>
     <pointsize>16</pointsize>
     <italic>true</italic>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color:rgb(85, 170, 255)</string>
   </property>
   <property name="text">
    <string>TELA DE GERENCIAMENTO</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_8">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>641</width>
     <height>711</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap>fundo.jpg</pixmap>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="bConfirmar1">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>130</y>
     <width>221</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial</family>
     <pointsize>11</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color:rgb(255, 255, 255);
background-color:rgb(85, 170, 255)</string>
   </property>
   <property name="text">
    <string>CADASTRAR CLIENTE</string>
   </property>
  </widget>
  <widget class="QPushButton" name="bConfirmar2">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>200</y>
     <width>221</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial</family>
     <pointsize>11</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color:rgb(255, 255, 255);
background-color:rgb(85, 170, 255)</string>
   </property>
   <property name="text">
    <string>EDITAR CLIENTE</string>
   </property>
  </widget>
  <widget class="QPushButton" name="bConfirmar3">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>270</y>
     <width>221</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial</family>
     <pointsize>11</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color:rgb(255, 255, 255);
background-color:rgb(85, 170, 255)</string>
   </property>
   <property name="text">
    <string>PESQUISAR CLIENTE</string>
   </property>
  </widget>
  <widget class="QPushButton" name="bConfirmar5">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>410</y>
     <width>221</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial</family>
     <pointsize>11</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color:rgb(255, 255, 255);
background-color:rgb(85, 170, 255)</string>
   </property>
   <property name="text">
    <string>EXCLUIR CLIENTE</string>
   </property>
  </widget>
  <widget class="QPushButton" name="bConfirmar6">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>680</y>
     <width>21</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>PushButton</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnListarClientes">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>340</y>
     <width>221</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial</family>
     <pointsize>11</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color:rgb(255, 255, 255);
background-color:rgb(85, 170, 255)</string>
   </property>
   <property name="text">
    <string>LISTAR CLIENTES</string>
   </property>
  </widget>
  <zorder>bConfirmar6</zorder>
  <zorder>label_8</zorder>
  <zorder>label</zorder>
  <zorder>bConfirmar1</zorder>
  <zorder>bConfirmar2</zorder>
  <zorder>bConfirmar3</zorder>
  <zorder>bConfirmar5</zorder>
  <zorder>btnListarClientes</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
